### PR TITLE
feat: LT-2274 and LT-2275

### PR DESCRIPTION
**Pull request by andytaylorkainos**

### CODE REVIEW

File: applications/forms/end_use_details.py [hunk 1]
```diff
@@ -1,25 +1,53 @@
...
+        title=EndUseDetails.INTENDED_END_USE,
+        questions=[
+            TextArea(
+                name="intended_end_use",
+                short_title=EndUseDetails.EndUseDetailsSummaryList.INTENDED_END_USE,
+                extras={"max_length": 2200},
+                optional=False,
+            )
+        ],
+        default_button_name="Save and continue",
```
Comment (jfaracik) [inline applications/forms/end_use_details.py]: Content ;)

**Comment by iichr** [comment]
e2e tests coming soon™️

File: applications/helpers/task_lists.py [hunk 2]
```diff
@@ -67,15 +67,16 @@ def get_application_task_list(request, application, errors=None):
...
         context["countries"] = get_application_countries(request, application["id"])
         context["end_use_details"] = get_end_use_details(application)
         context["goodstypes"] = get_application_goods_types(request, application["id"])
         if application.get("goods_types"):
             destination_countries = [goods_type["countries"] for goods_type in application.get("goods_types")][0]
             context["destinations"] = set([destination["id"] for destination in destination_countries])
+    elif application_type == F680:
+        context["end_use_details"] = get_end_use_details(application)
     else:
         context["goods"] = get_application_goods(request, application["id"])
```
Comment (Mark-Spiers-kainos) [inline applications/helpers/task_lists.py]: I believe this stuff is still needed for standard applications etc, so can you double check if works fine?

Comment (andytaylorkainos) [reply applications/helpers/task_lists.py]: Standard and open logic is handled a few lines above :)

Comment (Mark-Spiers-kainos) [reply applications/helpers/task_lists.py]: but this code would have ran for standard before, it doesn't now. so can you just double check this doesn't affect anything

Comment (andytaylorkainos) [reply applications/helpers/task_lists.py]: I'll have another look :)

Comment (andytaylorkainos) [reply applications/helpers/task_lists.py]: Seems to all be handled :)

File: applications/forms/end_use_details.py (hunk 1 above)
Comment (andytaylorkainos) [reply applications/forms/end_use_details.py]: doh!

Comment (iichr) [reply applications/forms/end_use_details.py]: sorted

File: ui_automation_tests/features/generated_documents.feature [hunk 3]
```diff
@@ -6,6 +6,8 @@ So that I can download the documentation
   @LT_1552_generated_documents @regression
   Scenario: view Generated Documents on an application
     Given I go to exporter homepage and choose Test Org
-    When I go to the recently created application with a Generated Document attached
+    And I create a standard application via api
```
Comment (phlptt) [inline ui_automation_tests/features/generated_documents.feature]: 👍

**Pull request merged**